mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -488,11 +503,20 @@ def create_model_version(
         headers, run = self._get_run_and_headers(run_id)
         source_workspace_id = self._get_workspace_id(headers)
         notebook_id = self._get_notebook_id(run)
+        job_id = self._get_job_id(run)
+        job_run_id = self._get_job_run_id(run)
         extra_headers = None
-        if notebook_id is not None:
-            notebook_entity = Notebook(id=str(notebook_id))
-            entity = Entity(notebook=notebook_entity)
-            lineage_header_info = LineageHeaderInfo(entities=[entity])
+        if notebook_id is not None or job_id is not None:
+            entity_list = []
+            if notebook_id is not None:
+                notebook_entity = Notebook(id=str(notebook_id))
+                entity_list.append(Entity(notebook=notebook_entity))
+            if job_id is not None:
+                job_entity = Job(id=str(job_id), job_run_id=str(job_run_id))
+                if job_run_id is None:
+                    job_entity = Job(id=str(job_id))
```

Comment:
```suggestion
```
Maybe good to check but I think this is unnecessary, since the default value of `job_run_id` should already be `None`

Reply:
Yeah but:

`>>> x = str(None)
>>> x
'None'
>>> y = None
>>> y
>>> 
`

Reply:
Unless you are saying that job_run_id is never filled in.

Reply:
Do we actually need to cast to str? Or can we just pass the value of the tag (since MLflow run tag values are strings)

Reply:
Ah.  I missed the comment in run_tag.py that they were strings. (stupid python with it's lax typing).  I'll change and send out to review from others.